ebi_eva_common_pyutils/taxonomy/taxonomy.py

```diff
@@ -36,17 +37,44 @@ def normalise_taxon_scientific_name(taxon_name):
     return re.sub('[^0-9a-zA-Z]+', '_', taxon_name.lower())
 
 
-def get_normalized_scientific_name_from_ensembl(taxonomy_id: int) -> str:
+def get_normalized_scientific_name(taxonomy_id):
     """Get the scientific name for that taxon"""
-    return normalise_taxon_scientific_name(get_scientific_name_from_ensembl(taxonomy_id))
+    return normalise_taxon_scientific_name(get_scientific_name_from_taxonomy(taxonomy_id))
 
+def get_scientific_name_from_eva(taxonomy_id, private_config_xml_file, profile):
+    from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+    query = f"""select scientific_name from evapro.taxonomy where taxonomy_id={taxonomy_id}"""
+    with get_metadata_connection_handle(profile, private_config_xml_file) as pg_conn:
+        result = get_all_results_for_query(pg_conn, query)
+        if not result:
+            return None
+        else:
+            return result[0][0]
 
-def get_scientific_name_from_taxonomy(taxonomy_id: int) -> str:
+def get_scientific_name_from_eva_using_metadata_connection(metadata_connection_handle, taxonomy_id):
```

Comment:
```suggestion
def get_scientific_name_from_eva_using_metadata_connection(taxonomy_id, metadata_connection_handle):
```

Reply:
to be consistent with the order in the previous method

Reply:
updated